influxdb/2.0/alpine/entrypoint.sh

```diff
@@ -1,26 +1,17 @@
-#!/bin/sh
```

Comment:
This script is now identical to `influxdb/2.0/entrypoint.sh`. I considered rearranging the whole `2.0` directory so the 2 Dockerfiles were placed side-by-side (allowing us to delete the duplicate files). I decided against it for now because I didn't want to have 1 directory be different from the rest of the repository.